src/sentry/static/sentry/app/actionCreators/globalSelection.tsx

```diff
@@ -80,9 +222,7 @@ export function updateProjects(
     return;
   }
 
-  if (!router) {
-    GlobalSelectionActions.updateProjects(projects);
-  }
+  GlobalSelectionActions.updateProjects(projects);
```

Comment:
These action creators now *always* create actions to update the store as well as update the URL. Previously, it would update URL and the component would update store.